docs/admin/src/widgets/genericPreviewTemplate.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { ReactLiquid, liquidEngine } from 'react-liquid';
+import marked from 'marked';
+import template from '../../../_includes/generic-content.html';
```

Comment:
🤔 this path

Reply:
Hmmm yeah maybe I'll move that filepath prefix into a const called something like `JEKYLL_TEMPLATE_LOCATION` to clarify why it needs to reach into such a distant directory.